spec/features/admin/orders/admin_can_view_an_order_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+feature 'When an admin visits an order page' do
+
+  background do
+    allow_any_instance_of(ApplicationController)
+      .to receive(:current_user)
+      .and_return(create(:admin))
+
+    order = create(:order)
+    items = create_list(:item, 2, price: 7)
+    create(:order_item, order: order, quantity: 1, item: items.first)
+    create(:order_item, order: order, quantity: 2, item: items.last)
+    visit admin_order_path(order)
+  end
+
+  scenario 'they can see the order\'s date and time' do
+    expect(page).to have_content(Order.first.created_at)
```

Comment:
We might want to test for date and time independently, especially since we will probably need to format